index.js

```diff
@@ -31,11 +31,12 @@ var FeatureService = function (url, options) {
   this.options = options
   this.layer = options.layer || 0
   this.timeOut = 1.5 * 60 * 1000
+  var concurrency = this.url.split('//')[1].match(/^service/) ? 16 : 4
```

Comment:
Still going to need regex to know if it's a hosted feature service.

eg:
services3.arcgis.com
services1.arcgis.com